### ISSUE TITLE
is this library supporting tags in the metrcis?

### ISSUE THREAD

**Issue by josuepr2** [marpaia/graphite-golang#25]
Hi, I'm working with tags in my metrics but I'm not sure if this library support that ?

**Comment by marpaia** [comment]
I don't believe so. Do you have any docs on this feature?

**Comment by josuepr2** [comment]
yeah. the next link include information about of the 'tags' usage https://graphite.readthedocs.io/en/latest/tags.html
